src/index.js

```diff
@@ -1,5 +1,6 @@
 // @flow
-export { default as AppDevRouter} from './AppDevRouter';
 export { default as AppDevAPI }  from './AppDevAPI';
-export { default as RegisterSession } from './RegisterSession';
+export { default as AppDevRouter} from './AppDevRouter';
```

Comment:
consistency: either spaces or no spaces between braces (i prefer no spaces personally, we should write an appdev style guide bc airbnb has some questionable recommendations)

Reply:
nah we should have spaces

Reply:
in that case `AppDevRouter }`